pkg/ipcache/ipcache_test.go

```diff
@@ -0,0 +1 @@
+package ipcache
```

Comment:
File added by accident?

Reply:
No, this is a unit test for the new cache.

Reply:
Oops, I see what you meant by 'file added by accident'; I'll squash these commits together.